app/javascript/lib/vue-components/modules/SkeletonSpinner.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <template v-for="line in parseInt(lines)">
+      <div
+        :key="`line-${line}`"
+        :style="stylesLines()"
+        class="gobierto-skeleton-spinner gobierto-skeleton-spinner-line"
+      />
+    </template>
+    <template v-for="square in parseInt(squaresRows)">
+      <div
+        :key="`square-row-${square}`"
+        class="gobierto-skeleton-spinner-container-squares"
+      >
+        <template v-for="square in parseInt(squares)">
+          <div
+            :key="`square-${square}`"
+            :style="styleSquares"
+            class="gobierto-skeleton-spinner gobierto-skeleton-spinner-square"
+          />
+        </template>
+      </div>
+    </template>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "SkeletonSpinner",
+  props: {
+    heightSquare: {
+      default: `1rem`,
+      type: String,
+    },
+    squaresRows: {
+      default: '1',
+      type: String,
+    },
+    squares: {
+      default: '0',
+      type: String,
+    },
+    lines: {
+      default: '0',
+      type: String,
+    }
+  },
+  computed: {
+    styleSquares() {
+      return {
+        //
+        width: ((100 - (this.squares - 1)) / this.squares) + '%',
+        height: this.heightSquare
+      }
+    },
+  },
+  methods: {
+    stylesLines() {
```

Comment:
Why this method is not a computed one?

Reply:
It was. But this method determines the width of the lines, if it were computed the width would always be the same.